usr/bin/i2pbrowser

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+##
+##
+
+export SCRIPTNAME="$(basename "$BASH_SOURCE")"
+
+ICON="/usr/share/icons/anon-icon-pack/tbupdate.ico"
+
+if [ ! -d "$HOME/.i2pb/" ]; then
+    echo "Error, i2p browser does not appear to be installed(run update-i2pbrowser)"
+    exit 1
+fi
+
+if [ ! -d "$HOME/.i2pb/profile.i2p" ]; then
+    cp -r /usr/share/tb-profile-i2p/profile.i2p \
```

Comment:
This could fail silently due to access rights being messed up. Happens sometimes when users restore backups. I think /usr/bin/torbrowser woulnd't fail silently in such cases and show a proper error message.

Not a blocker.